src/Form/Form.js

```diff
@@ -686,19 +690,19 @@ class Form {
             }
         }
 
+        const isMobileApp = this.device.globalConfig.isMobileApp
         if (!(input instanceof HTMLSelectElement)) {
             const events = ['pointerdown']
-            if (!this.device.globalConfig.isMobileApp) events.push('focus')
+            if (!isMobileApp) events.push('focus')
             input.labels?.forEach((label) => {
-                if (this.device.globalConfig.isMobileApp) {
-                    // On mobile devices we don't trigger on focus, so we use the click handler here
-                    this.addListener(label, 'pointerdown', handler)
-                } else {
-                    // Needed to handle label clicks when the form is in an iframe
-                    this.addListener(label, 'pointerdown', handlerLabel)
-                }
```

Comment:
these comments contain important information - can you try to incorporate them, perhaps by moving the ternary up a line.